.github/workflows/ci.yml

```diff
@@ -41,6 +56,21 @@ jobs:
             os: ubuntu-latest
             toxenv: docs
 
+          - name: Bandit security audit
+            python-version: 3.9
+            os: ubuntu-latest
+            toxenv: bandit
```

Comment:
Bandit checks Python code only. I wonder if we should enable CodeQL for this repository.

Reply:
If so, then we should probably do it for asdf itself too.